### ISSUE TITLE
port calc-accessibility script into haddock3-restraints

### ISSUE THREAD

**Issue by mgiulini** [haddocking/haddock3#818]
the haddock-tools script calc-accessibility.py should be ported within haddock3 as it will be used for future tutorials
